README.md

```diff
@@ -68,7 +69,8 @@ The Access Key is contained in credentials.csv.
 
 **Note well:** credentials.csv contains `secrets`!
 Do not share them or store them in git.
-The [.gitignore](.gitignore) in this repository explicitly ignores this file.
+The [.gitignore](.gitignore) in this repository explicitly ignores this file. Altering the [.gitignore](.gitignore) file
+in this repository could result in your AWS credentials (credentials linked to *your* credit card information) being visible on Github. *NEVER COMMIT SECRETS TO GIT*
 
```

Comment:
Given that I replied to #9 with the word "objection" I'm not sure why this is here. :robot: :confused: 

Perhaps this and other hand-holding is necessary.  I just think that at this point in WDI, reading directions should be something the developers can handle.